packages/backend/src/peripherals/database/shared/types.ts

```diff
@@ -82,7 +82,7 @@ declare module 'knex/types/tables' {
     report_status: ReportStatusRow
     events: EventRow
     transactions: {
-      rpc: RpcTransactionCountRow
+      block: BlockTransactionRow
```

Comment:
Which one do you actually prefer? RN we have a bit inconsistency in repo, so that's probably a good point to bring it back. (we have `ZksyncTransactionRepository` and `StarkexTransactionCountRepository`). Personally I like the former as it's shorter. Or maybe it'd be best to say `BlockActivityRow`. That would be consistent with the module naming and it's most explanatory and the shortest. WYT?